src/cargo/ops/cargo_new.rs

```diff
@@ -955,8 +963,52 @@ fn update_manifest_with_inherited_workspace_package_keys(
 fn update_manifest_with_new_member(
     root_manifest_path: &Path,
     workspace_document: &mut toml_edit::Document,
-    package_path: &Path,
+    display_path: String,
```

Comment:
nit: to make the caller more refactor friendly, this should probably be a `&str`